### PR TITLE
docs: update typescript node & browser generator source code links

### PR DESCRIPTION
**Pull request by minaelee**
per move to monorepo

### CODE REVIEW

**Comment by dannysheridan** [comment]
can you double check these version numbers are the latest?

File: fern/docs/pages/sdks/browser-sdk.mdx [hunk 1]
```diff
@@ -31,7 +30,7 @@ groups:
   internal:
     generators:
       - name: fernapi/fern-typescript-browser-sdk
-        version: 0.8.13
+        version: 0.9.6
```
Comment (minaelee) [inline fern/docs/pages/sdks/browser-sdk.mdx:33]:
```suggestion
        version: 0.11.5
```

File: fern/docs/pages/sdks/browser-sdk.mdx [hunk 2]
```diff
@@ -46,7 +45,7 @@ groups:
   publish:
     generators:
       - name: fernapi/fern-typescript-browser-sdk
-        version: 0.9.5
+        version: 0.9.6
```
Comment (minaelee) [inline fern/docs/pages/sdks/browser-sdk.mdx:48]:
```suggestion
        version: 0.11.5
```

**Comment by minaelee** [comment]
updating in different PR

**Pull request closed without merging**